src/cmd/cli/command/commands.go

```diff
@@ -478,9 +478,20 @@ var generateCmd = &cobra.Command{
 				}
 				if sample == generateWithAI {
 					sample = ""
-					defaultFolder = "project1"
-				} else {
-					defaultFolder = sample
+				}
+			}
+		}
+
+		defaultFolder := sample
+		if defaultFolder == "" {
+			// Find the first available project folder name; TODO: can we ask AI for a suggestion?
```

Comment:
The old trick of selecting a random adjective and noun from a dictionary could get us pretty far for generating a sample project name without needing to get AI in the mix.

Reply:
My idea was to have the idea generate the code *and* provide a project name, based on the same prompt, all within the same API call.